### PR TITLE
WelcomePage cmp and graphs responsivity

### PR DESCRIPTION
**Pull request by zoyar-moveo**
hey :)

I didn't have the time to continue with the data catching from the API so I commented it out

### CODE REVIEW

File: src/GlobalStyles.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default createGlobalStyle`
 }
 body{
     overflow: hidden;
+    font-size : 16px;
```
Comment (noam-movhls) [inline src/GlobalStyles.ts]: Is the resolution issued fixed?

File: src/components/DataCard/charts/sourcesChart/SourcesChart.tsx [hunk 2]
```diff
@@ -17,7 +17,8 @@ export interface chartProps {
 
 const SourcesChart: React.FC<chartProps> = (props) => {
   return (
-    <div style={{ width: "150px", height: "150px" }}>
+    // <div style={{ width: "300px", height: "300px" }}>
```
Comment (noam-movhls) [inline src/components/DataCard/charts/sourcesChart/SourcesChart.tsx]: redundant?

File: src/components/FeedCard/FeedCard.tsx [hunk 3]
```diff
@@ -48,7 +47,7 @@ const FeedCard: React.FC<FeedCardProps> = ({ feedCardObj, isMobile }) => {
   return (
     <CardContainer>
       <Img alt="" src={feedCardObj.urlToImage} />
-      <FlexContainer>
+      <ContentContainer>
         <div className="flex row space-between">
```
Comment (noam-movhls) [inline src/components/FeedCard/FeedCard.tsx]: We prefer to use one approach for styling, so it's better to create Style Components for that

File: src/GlobalStyles.ts (hunk 1 above)
Comment (zoyar-moveo) [reply src/GlobalStyles.ts]: not sure

**Pull request merged**